src/twisted/internet/base.py

```diff
@@ -629,6 +629,52 @@ def __init__(self) -> None:
             self._initThreads()
         self.installWaker()
 
+    # Signal handling pieces
+    _installSignalHandlers = False
+
+    def _handleSignals(self) -> None:
+        """
+        Install the signal handlers for the Twisted event loop.
+        """
+        try:
+            import signal
+        except ImportError:
+            log.msg(
```

Comment:
Is this configuration even possible to achieve any more?

Reply:
Also, TODO for a later PR here, it's killing me that we have all these `log.msg` calls everywhere. 😬